deployment/README.md

```diff
@@ -209,14 +211,19 @@ as directed below.
 
 #### Storage
 
-The deployer initially creates an ephemeral deployment in which all
+By default, the deployer creates an ephemeral deployment in which all
 of a pod's data will be lost any time it is restarted. For production
 use you should specify a persistent storage volume for each deployment
-of ElasticSearch. Use the `oc volume` command for adding volumes to
-deployments.
+of ElasticSearch. The deployer parameters with `PVC` in the name should
+be used for this. You can either use an existing set of PVCs (by naming
+them all in a series starting with a prefix and appending numbers starting
+at 1), or have them created requesting a specified size. This is the
```

Comment:
The complexity is that they can choose a different prefix if they want for some reason. But yeah, an example from the default would be good.